.github/ISSUE_TEMPLATE/Bug_report.yml

```
@@ -0,0 +1,33 @@
+name: Bug Report
+description: Create a report to help us improve SciPy
+title: "[Bug]: "
+labels: [defect]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a bug report.
+  - type: markdown
+    attributes:
+        value: Please fill in the fields below, deleting the sections that don't apply to your issue.
+  - type: markdown
+    attributes:
+        value: You can view the final output by clicking the preview button above.
+  - type: input
+    attributes:
+      label: Reproducing Code Example
+      description: 'If you place your code between the triple backticks below, it will be rendered as a code block.'
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Error message
+      description: 'If any, paste the *full* error message inside a code block as above (starting from line Traceback)'
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Scipy/Numpy/Python version information
+      description: 'You can simply run the following and paste the result in a code block - `import sys, scipy, numpy; print(scipy.__version__, numpy.__version__, sys.version_info)`'
```

```suggestion
      description: 'Please run the following and paste the result in a code block - `import sys, scipy, numpy; print(scipy.__version__, numpy.__version__, sys.version_info)`'
```
("simple" is a good word to avoid IMO, for someone new, nothing is simple.)

---

Done!